initialize-local-db.sql

```diff
@@ -0,0 +1,5 @@
+DROP DATABASE if exists food_truck_db;
+CREATE DATABASE food_truck_db;
+
+DROP USER if exists admin_user;
+CREATE ROLE admin_user WITH LOGIN PASSWORD '1234';
```

Comment:
What is this for ?

Reply:
This is our script for sharing the database between multiple people, it runs when you follow the steps on the README